core/finality-grandpa/src/tests.rs

```diff
@@ -157,6 +157,21 @@ fn make_topic(round: u64, set_id: u64) -> Hash {
 	hash
 }
 
+fn make_commit_topic(set_id: u64) -> Hash {
+	let mut hash = Hash::default();
+
+	{
+		let raw = hash.as_mut();
+		raw[16..22].copy_from_slice(b"commit");
```

Comment:
Are we sure that the hash has always enough bytes?

Reply:
in tests, that seems like a reasonable assumption